lib/crazy_eight_game.rb

```diff
@@ -0,0 +1,161 @@
+require 'JSON'
+require 'rest-client'
+require 'pry'
+
+class CrazyEightGame < ActiveRecord::Base
+
+    def new_deck
+        self.deck_id = JSON.parse(RestClient.get("https://deckofcardsapi.com/api/deck/new/shuffle/"))['deck_id']
+        update_remaining(JSON.parse(RestClient.get("https://deckofcardsapi.com/api/deck/new/shuffle/"))['remaining'])
+        save
+    end
+
+    def draw_from_api(count)
+        response = RestClient.get("https://deckofcardsapi.com/api/deck/#{deck_id}/draw/?count=#{count}")
+        update_remaining(JSON.parse(response)['remaining'])
+        JSON.parse(response)['cards']
+    end
+
+    def deal_start_hand(player)
+        draw_from_api(7).each do |c| 
```

Comment:
```suggestion
        draw_from_api(7).each do |card| 
```
Generally its bad practice to shorten any variable length to less than 3 characers, unless it has a specific convention. Ex. x,y for coordinates, or i for an index variable.